app/components/BoxShadow.tsx

```diff
@@ -72,6 +72,22 @@ const $offsetPresets = {
     },
   ] as StyleProp<ImageStyle>,
 
+  secondary: [
+    {
+      tintColor: colors.palette.secondary500,
+      borderColor: colors.palette.neutral300,
+      borderWidth: 1,
+    },
+  ] as StyleProp<ImageStyle>,
+
+  reversed: [
```

Comment:
Don't know if there are better names for these 😅

Reply:
Reversed makes sense to me!